flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -261,6 +261,12 @@
      */
     private volatile boolean isRunning;
 
+    /**
+     * Flag to mark the task "in operation", in which case check needs to be process the exceptions
```

Comment:
It's a little obscure here
1. What is the meaning of `in operation`?
2. IMO, we'd better describe when the `StreamTask` will be in this state, rather than say that this state is for handling exception.

Reply:
> 1. What is the meaning of `in operation`?

It would be better to clarify the `in operation` to `in Initializing operation` or `in restoring operation`

Reply:
> 2\. IMO, we'd better describe when the `StreamTask` will be in this state, rather than say that this state is for handling exception.

SGTM.